core/conversion/converters/impl/element_wise.cpp

```diff
@@ -359,8 +359,22 @@ auto element_wise_registrations TRTORCH_UNUSED =
                     // Should implement self * other
                     auto self = args[0].ITensorOrFreeze(ctx);
                     auto other = args[1].ITensorOrFreeze(ctx);
-                    auto mul =
+                    nvinfer1::ILayer* mul = nullptr;
```

Comment:
@inocsin Are these changes that would only be relevant to mul or other ops in general? Also did you write tests for this? I didnt see them in the commits

Reply:
Currently, I only find the Bool * Int operation in my models. I think other operation like Int / Bool or Int +/- Bool doesn't make  any sense. I will add test case later.

Reply:
The input type of trtorch is set to float in default, I have to modify the TRTorch/core/conversion/conversion.cpp:150 
```
auto trt_in = ctx->net->addInput(name.c_str(), ctx->input_type, dims.input_shape);
```
to support input type of Bool

Reply:
we can try this https://github.com/NVIDIA/TRTorch/pull/327

Reply:
What does it mean to do bool * int? should that give you a bool out? like what does False * 8 mean?

Reply:
[True, False] * [8, 8] = [8, 0], this can be a mask operation, this demo graph here https://github.com/NVIDIA/TRTorch/pull/327

Reply:
we can close this PR if we merge https://github.com/NVIDIA/TRTorch/pull/327